docker/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  server:
+    build: ./server
+    image: "iwantbot/server"
+    volumes:
+     - ..:/iwant-bot-mount
+    ports:
+     - "8080:8080"
+  database:
+    build: ./database
```

Comment:
The postgres container is not used so far. Judging from the image description (https://hub.docker.com/_/postgres/), I thought that we will end up with our Dockerfile extending it. I suppose that we will clarify this on our upcoming mentoring session.